Implement the Artwork Page

matheus23 opened the pull request:
As discussed with @bmann , I implemented an artwork page.

Look at it live here: https://huge-junior-round-wizard.fission.app/artwork/

matheus23 commented on package.json:
```diff
@@ -3,24 +3,26 @@
     "dev": "elm-pages develop",
-    "build": "elm-pages build"
+    "build": "elm-pages build",
+    "up-deps": "elm-json upgrade --unsafe && npm up && npm outdated"
```
Haha Whoops. Yeah I'm used to using yarn.
There are too many package managers in the javascript ecosystem :)

I'll figure out just. Makes sense to put everything there.

icidasset replied:
Yeah, I used yarn previously as well.
But then I hit this edge-case with github dependencies and I had to switch to something else 🤷

matheus23 replied:
Yeah. I just checked it, pnpm doesn't work with elm-pages.

>  But then I hit this edge-case with github dependencies and I had to switch to something else :shrug: 
Yarn does support github dependencies, now at least: https://classic.yarnpkg.com/en/docs/cli/add/
> 5. yarn add <git remote url> installs a package from a remote git repository.

Building and running the project with yarn works.

The package manager ecosystem is in a weird state right now. Yarn and npm now do dependency deduplication, solving an issue which pnpm pioneered solving. And git dependencies are now supported by all of them, too. :man_shrugging: 

I'll keep it at npm for now, though.

matheus23 replied:
I just noticed, the yarn.lock file is on master, too :D

matheus23 commented on src/Matter/Artwork.elm:
```diff
@@ -0,0 +1,425 @@
+                    , T.py_1
+                    , T.rounded_lg
+                    , T.text_gray_600
+                    , T.md__text_lg
```
Ah that explains it.

Yeah, I've seen the 'Responsive' comments. Alright.

It's possible to create an elm-review rule for alphabetized class order :thinking: (Definitely way too much effort, but kinda cool)

Should we create a styleguide somewhere?

icidasset replied:
An elm-review rule could be cool, but not sure how that'd would work (you'd have split up into sections based on comments?). Would be super cool if it could auto fix based on said created rule 💯 

We can do a small style guide for sure 👍 
Let me know if you'd like me to do that, or if you wanna do that.

bmann replied:
Let's put as much as possible on to @matheus23 -- and I'll just file it as an issue here for now and we can prioritize as we go.

matheus23 commented on src/Matter/Artwork.elm:
```diff
@@ -0,0 +1,425 @@
+            [ Html.h1
+                [ -- TODO this is actually supposed to be Work Sans, so font_display,
+                  -- but we have to make a thinner version available somehow.
+                  -- Karla on the other hand is a little thinner by default
```
Thanks!

matheus23 commented:
I addressed the reviews and added some finishing touches.

I've also added a "Learn more" button to the top right, like on the support page. I thought that'd be more consistent. Let me know what you think.

I think this is ready for merging :)
